### PR TITLE
(CU-a0ujk4) added: full changelog and three versions

### PR DESCRIPTION
**Pull request by ivan-deity**
Task: https://app.clickup.com/t/a0ujk4

Under Release Notes:
- Added changelogs for three versions: 2.4.6/2.4.8/2.5.0
- Added our full changelog

image

### CODE REVIEW

File: docs/platform/release/2-4-6.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+id: 2-4-6
```
Comment (Borales) [inline docs/platform/release/2-4-6.md:2]: could we also add dates for the release pages?

**Review by Borales — APPROVED** [review]

File: docs/platform/release/2-4-6.md (hunk 1 above)
Comment (ivan-deity) [reply docs/platform/release/2-4-6.md:2]: I added them to the title, I hope that is what you meant.

Comment (Borales) [reply docs/platform/release/2-4-6.md:2]: @ivan-deity I'd prefer to have it more subtle (not as a part of the page title, so maybe as a smaller subtitle), but I'd imagine that would require more work for the templates - so it's fine

**Review by Borales — APPROVED** [review]
Overall - 👍 

The only complaint is about not having documented the new features like https or ioc, but that wasn't a part of this PR.

File: docs/platform/release/2-4-6.md (hunk 1 above)
Comment (ivan-deity) [reply docs/platform/release/2-4-6.md:2]: @Borales, it's not a problem, I will try to add it today, no worries :))

**Review by hobadams — APPROVED** [review]
Thanks so much for this

**Pull request merged**